### PR TITLE
DOC-2537 Doc for social sharing on mobile

### PR DESCRIPTION
**Pull request by catong**
DOC-2537
This PR adds documentation to the Learner's Guide for the ability to share course links and badges on the mobile apps. Applies to Open edX learners only.

Draft HTML is here: 
http://draft-mobile-social-share.readthedocs.io/en/latest/SFD_certificates.html
http://draft-mobile-social-share.readthedocs.io/en/latest/SFD_mobile.html

REVIEWERS:
@aleffert 
@lamagnifica 
FYI @natabene

### CODE REVIEW

**Comment by aleffert** [comment]
👍

**Comment by catong** [comment]
Squashed commits

File: en_us/shared/students/SFD_mobile.rst [hunk 1]
```diff
@@ -82,6 +82,26 @@ but not all, of the problems in your assignments. To complete an entire
 course, including participating in course discussions, you use a web browser
 on a computer.
 
+==================================================
+How do I share a link to a course I am taking?
+==================================================
+
+After you enroll in a course, you can let family and friends know about the
+course by sharing a link to it on a connected social account, or in email.
+.
```
Comment (lamagnifica) [inline en_us/shared/students/SFD_mobile.rst]: stray period?

File: en_us/shared/students/SFD_mobile.rst [hunk 2]
```diff
@@ -82,6 +82,26 @@ but not all, of the problems in your assignments. To complete an entire
 course, including participating in course discussions, you use a web browser
 on a computer.
 
+==================================================
+How do I share a link to a course I am taking?
+==================================================
```
Comment (lamagnifica) [inline en_us/shared/students/SFD_mobile.rst]: is this topic for all mobile users, or open edX only? need include directive?

File: en_us/shared/students/SFD_mobile.rst (hunk 1 above)
Comment (lamagnifica) [inline en_us/shared/students/SFD_mobile.rst]: ...in ^an email ^message.
...by sending an email message. (?)

File: en_us/shared/students/SFD_mobile.rst [hunk 3]
```diff
@@ -82,6 +82,26 @@ but not all, of the problems in your assignments. To complete an entire
...
+From a course's main page, select the **Share** icon in the course's title
+display.
+
+  .. image:: ../../shared/students/Images/SFD_Mobile_Course_Share.png
+    :width: 300
+    :alt: A course page on the mobile app, with the Share icon.
+
+On the page that opens, select how you want to share the link. For example, if
+you select Facebook, a post is created on your Facebook page with the link to
+the course included.
```
Comment (lamagnifica) [inline en_us/shared/students/SFD_mobile.rst]: this is a little convoluted... maybe just
For example,  select **Facebook** to post a link to the course on your Facebook wall.

File: en_us/open_edx_students/source/SFD_certificates.rst [hunk 4]
```diff
@@ -27,16 +27,19 @@ for the course platform that you are taking a course on.
 Accessing a Certificate
 ***********************
 
-Some courses issue web-based certificates. You can access a web-based
-certificate from your dashboard.
+Some courses issue certificates, which you can access from your dashboard.
 
-The following example from the edx.org website shows the **View Certificate**
-option on a learner dashboard.
+The following examples from the edx.org website shows the **View Certificate**
```
Comment (lamagnifica) [inline en_us/open_edx_students/source/SFD_certificates.rst]: shows --> show

Comment (lamagnifica) [inline en_us/open_edx_students/source/SFD_certificates.rst]: maybe strike "from the edx.org website"? repeated below

File: en_us/open_edx_students/source/SFD_certificates.rst [hunk 5]
```diff
@@ -56,6 +59,13 @@ You can share your web certificates on Facebook, LinkedIn, and Twitter.
    A dialog box for the social site you selected opens, with the certificate
    URL already entered.
```
Comment (lamagnifica) [inline en_us/open_edx_students/source/SFD_certificates.rst:61]: does this need an include directive?

File: en_us/open_edx_students/source/SFD_certificates.rst (hunk 4 above)
Comment (catong) [inline en_us/open_edx_students/source/SFD_certificates.rst]: Yes, thanks

File: en_us/open_edx_students/source/SFD_certificates.rst [hunk 6]
```diff
@@ -56,6 +59,13 @@ You can share your web certificates on Facebook, LinkedIn, and Twitter.
...
 
+   On the mobile app, select the **Share** icon to open a page from which you
```
Comment (lamagnifica) [inline en_us/open_edx_students/source/SFD_certificates.rst:62]: Kind of convoluted. Maybe make more task oriented, and assume a little more?
You can also share a web certificate from the mobile app. Select the **Share** icon, and then choose a social network.

File: en_us/open_edx_students/source/SFD_certificates.rst (hunk 5 above)
Comment (catong) [inline en_us/open_edx_students/source/SFD_certificates.rst:61]: Will investigate in a separate PR.

File: en_us/shared/students/SFD_mobile.rst (hunk 2 above)
Comment (catong) [inline en_us/shared/students/SFD_mobile.rst]: Yes, thanks; fixing

File: en_us/shared/students/SFD_mobile.rst (hunk 1 above)
Comment (catong) [inline en_us/shared/students/SFD_mobile.rst]: Yes

File: en_us/open_edx_students/source/SFD_certificates.rst [hunk 7]
```diff
@@ -56,6 +59,13 @@ You can share your web certificates on Facebook, LinkedIn, and Twitter.
...
+   choose the social network to share your certificate to.
+
+.. image:: ../../shared/students/Images/SFD_Mobile_SocialShare.png
```
Comment (lamagnifica) [inline en_us/open_edx_students/source/SFD_certificates.rst:65]: indent so it appears under the indented paragraph, then followed by a final step?

File: en_us/open_edx_students/source/SFD_certificates.rst [hunk 8]
```diff
@@ -107,20 +116,27 @@ badging site such as Mozilla Backpack.
 Share Badges From Your Profile
 ===============================
 
-To share badges from your profile on the Mozilla Backpack site, follow these
+To share badges from your profile to the Mozilla Backpack site, follow these
 steps.
```
Comment (lamagnifica) [inline en_us/open_edx_students/source/SFD_certificates.rst:120]: yay!

File: en_us/open_edx_students/source/SFD_certificates.rst [hunk 9]
```diff
@@ -107,20 +116,27 @@ badging site such as Mozilla Backpack.
...
 
+.. note:: You have to create an account on Mozilla Backpack before you can share
+   your badges.
+
 #. On your profile, select **Accomplishments**.
 
-#. For the badge that you want to share, select **Share**.
+#. For the badge that you want to share, select **Share**, or on the edX mobile
+   apps, tap the share icon.
```
Comment (lamagnifica) [inline en_us/open_edx_students/source/SFD_certificates.rst]: the **Share** icon.

File: en_us/open_edx_students/source/SFD_certificates.rst [hunk 10]
```diff
@@ -107,20 +116,27 @@ badging site such as Mozilla Backpack.
...
```
Comment (lamagnifica) [inline en_us/open_edx_students/source/SFD_certificates.rst:132]: preface these two images the way you did elsewhere, saying ones for the web and one's for mobile?

File: en_us/open_edx_students/source/SFD_certificates.rst [hunk 11]
```diff
@@ -107,20 +116,27 @@ badging site such as Mozilla Backpack.
...
     Accomplishments section of your user profile page.
 
+.. image:: ../../shared/students/Images/SFD_Mobile_Profile_Accomplishments.png
+   :width: 300
+   :alt: Choose the social network to which you want to share your certificate.
+
 You see instructions for downloading your badge and then sharing it on
-the Mozilla Backpack site. You have to create an account on Mozilla
-Backpack before you can share your badge.
+the Mozilla Backpack site.
```
Comment (lamagnifica) [inline en_us/open_edx_students/source/SFD_certificates.rst:142]: great.

**Comment by lamagnifica** [comment]
👍 made a few suggestions. Only confusing thing is that your description above indicates that these features are for Open edX only, but they aren't identified that way inline...

**Comment by catong** [comment]
@lamagnifica Thanks for the review. I applied the condition to the mobile file only in the "Course Questions" section. Conditions not necessary in the Certificates file because that file is exclusive to Open edX.

**Comment by catong** [comment]
Merging https://github.com/edx/edx-documentation/pull/1016

**Pull request merged**